.github/workflows/run-post-submission-jobs.yaml

```diff
@@ -20,11 +20,50 @@ permissions:
 
 jobs:
 
+  get-all-dates:
+    runs-on: ubuntu-latest
+    outputs:
+      NOWCAST_DATE: ${{ steps.set-dates.outputs.NOWCAST_DATE }}
+      SEQUENCE_AS_OF: ${{ steps.set-dates.outputs.SEQUENCE_AS_OF }}
+      matrix: ${{ steps.set-dates.outputs.MATRIX_DATES }}
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v4
+        with:
+            # only checkout what we need
+            sparse-checkout: |
+              .github/
+
+      - name: Set dates 🕰️
+        id: set-dates
+        # Nowcast_date = round_id of the round that just closed for submissions.
+        # Sequence_as_of = date used to get the sequences for clade assignment.
+        # Matrix_dates = dates used for matrix in create-target-data job (including
+        # the round_id of the 91-day-old round that can now be officially closed).
+        run: |
+          source ${GITHUB_WORKSPACE}/.github/shared/shared-functions.sh
+          nowcast_date=$(get_latest_wednesday "${{ inputs.nowcast_date }}")
+          sequence_as_of=$(date -d "$nowcast_date - 1 day" +%Y-%m-%d)
+          matrix_dates=$(generate_weekly_dates "$nowcast_date" 13)
+          echo "nowcast_date=$nowcast_date" >> $GITHUB_OUTPUT
```

Comment:
I didn't know about [$GITHUB_OUTPUT](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-output-parameter) in earlier iterations of this workflow--unlike GITHUB_ENVIRONMENT, the contents of GITHUB_OUTPUT can be passed between jobs. So now we can get all the dates at the beginning and share them as needed.